pypeit/display/__init__.py

```diff
@@ -30,3 +30,9 @@ def setup_SlitWavelength():
                  module='ginga_plugins', klass='SlitWavelength',
                  ptype='global', workspace='right', start=False,
                  category='PypeIt', menu='SlitWavelength', tab='SlitWavelength')
+
+def setup_Spec1dView():
+    return Bunch(path=os.path.join(os.path.split(__file__)[0], 'ginga_spec1dview.py'),
+                 module='ginga_spec1dview', klass='Spec1dView',
+                 ptype='local', workspace='right', start=False,
```

Comment:
Can you clarify the difference between `ptype='local'` and `ptype='global'`?  The `setup_SlitWavelength` uses `global`; should we change that to `local`?

Reply:
@kbwestfall, a local plugin can have an instance open for each channel in Ginga (a channel is an organizing device for a bunch of files that get cycled through a viewer in a window). A global plugin has only one instance open for all channels. So basically, a singleton instance for many viewers or one instance per viewer. Which you choose depends on how you are organizing your code and what it needs to do.  In this case, I'd prefer to track things by viewer, since we might have multiple plot viewers open. It's easier to do that by having it be a local plugin.